src/Page/Api.elm

```diff
@@ -33,7 +33,7 @@ type alias News =
 
 init : Session -> ( Model, Session, Cmd Msg )
 init session =
-    ( (), session, Ports.scrollTo { x = 0, y = 0 } )
+    ( (), session, Ports.loadRapidoc "/js/rapidoc-9.3.4.min.js" )
```

Comment:
We don't want/need the scrollTo anymore?

Reply:
Damn, you're fast. Fixed and rebased.